CMakeLists.txt

```diff
@@ -141,6 +141,60 @@ if(VALGRIND_TESTS)
 endif()
 
 #
+# cgreen
+#
+
+if(NOT CUKE_DISABLE_CGREEN)
+    find_package(cgreen)
+endif()
+
+#
+# Valgrind
+#
```

Comment:
The build errors are due to this section being duplicated (suspect during rebasing conflict).